docs/releases/v2023.1.1.rst

```diff
@@ -0,0 +1,49 @@
+Gluon 2023.1.1
+==============
+
+Important notes
+---------------
+
+Upgrades to v2023.1.1 and later releases are only supported from releases v2021.1 and later.
+This is due to migrations that have been removed to simplify maintenance.
+
+
+Bugfixes
+--------
+
+- x86: fix config loss during direct upgrades from v2021.1.x to v2023.1.x  (`#2972 <https://github.com/freifunk-gluon/gluon/pull/2972>`_)
+
+- tunneldigger: remove watchdog script that always failed and caused endless restarts (`#2987 <https://github.com/freifunk-gluon/gluon/pull/2987>`_)
```

Comment:
One more Tunneldigger fix: Tunneldigger was not using the WAN DNS server to resolve hostnames (regression in 2023.1)